### ISSUE TITLE
Verify Meshery compatibility with VirtualBox

### ISSUE THREAD

**Issue by leecalcote** [layer5io/meshery#111]
Will install Meshery in VirtualBox and update documentation based on experience.

Tasks:

1. Install Meshery locally. (using Meshery cli which runs all the services on `Docker` using `docker-compose`)
2. Deploy Kubernetes on minikube with VirtualBox provider (which is the default).
3. Configure Meshery with Kubernetes on minikube. [using the kubeconfig file mostly found in your $HOME/.kube/config (OR) by getting the config from `kubectl config view --raw > config` command]
3. Deploy each supported service mesh & a sample app in that environment using Meshery.
4. Performance test each supported service mesh in the minikube env using Meshery.
5. Update documentation with changes needed.

**Comment by girishranganathan** [comment]
@venkatasankarapu any progress here?

**Comment by venkatasankarapu** [comment]
@girishranganathan, Installed the k8s ( 3 nodes) and run the pods of meshery. will deploy the app and do the test.

**Comment by sakomws** [comment]
Successfully tested with Minikube:

K8S: v1.14.1
Minikube version: v1.0.0
Istio 1.1.5

Start minikube:
```
sudo minikube start --cpus 4 --memory 8192
```

Create config while current-context is minikube:
```
kubectl config view --minify --flatten > config.yaml
```

Follow Meshery standard flow.

**Comment by leecalcote** [comment]
@sakomws, that’s great news. No issue with access to localhost:9081? It didn’t need to be 192.168.1.100?

**Comment by sakomws** [comment]
Nope, works as charm, it is 192.168.1.100. Below is valid one, managed to deploy Istio to:
```
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: < cert shortcutted >
    server: https://192.168.99.100:8443
  name: minikube
contexts:
- context:
    cluster: minikube
    user: minikube
  name: minikube
current-context: minikube
kind: Config
preferences: {}
users:
- name: minikube
  user:
    client-certificate-data: <cert shortcutted >
    client-key-data: < key shortcutted >
```

**Comment by leecalcote** [comment]
> kubectl config view --minify --flatten > config.yaml

Excellent, @sakomws @venkatasankarapu. Doc update needed here.

**Comment by leecalcote** [comment]
@venkatasankarapu do we consider this issue complete or is there a documentation update needed? https://meshery.layer5.io/docs

**Comment by leecalcote** [comment]
Fixed with #122
